halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/services/v1/DaemonService.java

```diff
@@ -39,6 +37,12 @@ DeploymentConfiguration getDeployment(
       @Path("deployment") String deployment,
       @Query("validate") boolean validate);
 
+  @POST("/v1/config/deployments/{deployment}/generate/")
+  Void generateDeployment(
+      @Path("deployment") String deployment,
+      @Body String _ignore,
```

Comment:
Why are we catching the `@Body` if we're ignoring it? I think you can just exclude it.